### PR TITLE
gdcm 3.0.5

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/gdcm.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Gdcm < Formula
   desc "Grassroots DICOM library and utilities for medical files"
   homepage "https://sourceforge.net/projects/gdcm/"
-  url "https://github.com/malaterre/GDCM/archive/v3.0.4.tar.gz"
-  sha256 "9e460e5aae0082e5bb0cb4e6cb1293f4ea902394406e2260cc92674f7bd481b2"
+  url "https://github.com/malaterre/GDCM/archive/v3.0.5.tar.gz"
+  sha256 "5cc175d9b845db91143f972e505680e766ab814a147b16abbb34acd88dacdb5a"
 
   bottle do
     rebuild 1
```
Comment (SMillerDev) [inline Formula/gdcm.rb]: This can be removed now

Comment (zbeekman) [inline Formula/gdcm.rb]:
```suggestion
```

**Comment by zbeekman** [comment]
Should be ready to merge after CI finishes, but my commit via Github web UI should be fixed up.

**Comment by zbeekman** [comment]
merging 🍻

**Pull request closed without merging**